src/test/googleMock.ts

```diff
@@ -0,0 +1,54 @@
+import nock from 'nock';
+import { PEOPLE_BASE_URL } from '../lib/providers/googleProvider';
+import { GOOGLE_TOKEN, GOOGLE_USER_EMAIL } from './integration/google.test';
+
+const googleMock = nock(PEOPLE_BASE_URL).persist();
+
+export const getGoogleMock = () =>
+    googleMock
+        .get('/v1/people/me')
+        .query(searchParams => searchParams.access_token === GOOGLE_TOKEN)
+        .reply(200, {
```

Comment:
If you utilize the types (e.g. the GoogleField), you could type guard the mocks as well and would warn you when the interface change to update this

Reply:
types added